src/turbopuffer.test.ts

```diff
@@ -235,10 +253,9 @@ test("sanity", async () => {
     gotError = e;
   }
   expect(gotError).toStrictEqual(
-    new TurbopufferError(
-      "💔 Resource not found; the requested namespace is empty.",
-      { status: 404 },
-    ),
+    new TurbopufferError("🤷 namespace 'typescript_sdk_sanity' was not found", {
```

Comment:
change, wasn't passing, likely due to your work @morgangallant 👍🏻 better error for sure

Reply:
thanks for updating this!!